File: JavaScript/Searching/LinearSearch.js
function LinearSearch(l,v){
    
    for(let i = 0; i<l.length; i++){
        if(v == l[i]){
            return i
        }

    }
    return -1
}

list1 = [6,5,4,3,7,2]
val = 7;
console.log(LinearSearch(list1,val))

File: Python/Searching/linearSearch.py
def linearSearch(l,v):

    i = 0
    pos = -1
    for x in l:
        if x == v:
            pos = i
        i += 1
    return(pos)

a = [1,5,7,3,6,9]
b=9

print(linearSearch(a,b))
    
    
